src/components/common/Directors.vue

```diff
@@ -1341,70 +1288,32 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
     return (director.isDirectorActionable !== undefined) ? director.isDirectorActionable : true
   }
 
-  /** Called when there is a change (ie, some <-> none) in directors with a paid change. */
-  @Watch('directorsPaidChange')
```

Comment:
I simplified this by sending these events every time we emit an update (see new lines 1303-1304).